ddtrace/contrib/django/middleware.py

```diff
@@ -17,10 +18,18 @@
 
 log = logging.getLogger(__name__)
 
+def insert_exception_middleware():
+    exc_mdw = 'ddtrace.contrib.django.TraceExceptionMiddleware'
```

Comment:
can't we use more meaningful variables? `name`? `middleware_key`? `for key in middleware_keys` etc...
it's better not to short names so much otherwise it makes the code not so aesthetic :D